yarn-project/ethereum/src/config.ts

```diff
@@ -77,12 +77,12 @@ export const DefaultL1ContractsConfig = {
   aztecTargetCommitteeSize: 48,
   lagInEpochs: 2,
   aztecProofSubmissionEpochs: 1, // you have a full epoch to submit a proof after the epoch to prove ends
-  activationThreshold: BigInt(100e18),
-  ejectionThreshold: BigInt(50e18),
-  localEjectionThreshold: BigInt(98e18),
-  slashAmountSmall: BigInt(10e18),
-  slashAmountMedium: BigInt(20e18),
-  slashAmountLarge: BigInt(50e18),
+  activationThreshold: 100n * 10n ** 18n,
+  ejectionThreshold: 50n * 10n ** 18n,
+  localEjectionThreshold: 98n * 10n ** 18n,
```

Comment:
None of these values (except `slashAmountLarge`) matches what we have in the issue [TMNT-263: Rollup Configuration](https://linear.app/aztec-labs/issue/TMNT-263/rollup-configuration)) so don't think they are running any tests with the new values at all. (I had issue with that).

Reply:
If we change the defaults much here you will see an insane amount of tests failing because many of them are using hard coded values 😬